R/range-selector.R

```diff
@@ -56,7 +56,11 @@ dyRangeSelector <- function(dygraph,
   if (!is.null(dateWindow)) {
     if (length(dateWindow) != 2)
       stop("dateWindow must be vector of length 2 that is convertible to POSIXct")
-    selector$dateWindow <- sapply(USE.NAMES = FALSE, dateWindow, asISO8601Time)
+    # TODO: Rename dateWindow argument?
```

Comment:
Similar issue in terms of breaking code. Seeing that we have a second instance of date oriented existing syntax I'm inclined to leave the parameter values as-is but then have them "just work" for numeric graphs.